cli/src/main/java/com/devonfw/tools/ide/context/AbstractIdeContext.java

```diff
@@ -138,6 +138,9 @@ public abstract class AbstractIdeContext implements IdeContext {
 
   private WindowsHelper windowsHelper;
 
+  /** Context used for logging */
+  public static IdeContext loggingContext;
```

Comment:
please avoid public static (non-final + immutable) members.
Every class can now set this context.
Make this private and allow read-access via public static getter but avoid public write access.

Reply:
Set it to private and added a getter.